packages/wp-checkout/src/components/checkout-step.js

```diff
@@ -41,11 +42,15 @@ export default function CheckoutStep( {
 			<CheckoutStepHeader
 				stepNumber={ stepNumber }
 				title={ title }
+				completedTitle={ completedTitle }
 				isActive={ isActive }
 				isComplete={ isComplete }
 				onEdit={ onEdit }
 			/>
-			<StepContent>{ children }</StepContent>
+			<StepContent isActive={ isActive }>{ stepContent }</StepContent>
+			<StepSummary showSummary={ ( showSummary && ! isComplete ) || isComplete }>
```

Comment:
I think this logic could be simplified to just `showSummary={ isComplete || showSummary }`.

Reply:
We'll need to test this for the review step since we show the summary there when it's not completed.

Reply:
Ah... I think I understand. This is probably supposed to be `( showSummary && ! isActive ) || isComplete`. That would prevent `showSummary` from forcing the summary to display while the step is active.

🤔 It would be nice if we could come up with a way for the summary content _itself_ to decide if it is visible or not, rather than relying on the `CheckoutStep` wrapper. That way it would be easier to override for a custom review step or for a provider's billing contact component.

Reply:
> It would be nice if we could come up with a way for the summary content itself to decide if it is visible or not,

We could do this by making the `StepSummary` always visible when the step is not active, and then the summary component can decide what to display.

Reply:
I modified this in #36929. Let me know what you think.

Reply:
I like what you did there! Makes more sense that way.